common-lib/src/main/scala/com/gu/mediaservice/lib/aws/DynamoDB.scala

```diff
@@ -125,7 +128,42 @@ class DynamoDB[T](config: CommonConfig, tableName: String, lastModifiedKey: Opti
 
   def jsonGet(id: String, key: String)
              (implicit ex: ExecutionContext): Future[JsValue] =
-      get(id, key).map(item => asJsObject(item))
+    get(id, key).map(item => asJsObject(item))
+
+  def batchGet(ids: List[String])
```

Comment:
Should we move to [Scanamo](https://www.scanamo.org/batch-operations.html)?

Reply:
I'm not convinced that's a great use of Justin's time.